webpack.config.js

```diff
@@ -116,7 +116,7 @@ const config = {
 module.exports = [
     Object.assign({}, config, {
         entry: {
-            'app.bundle': './app.js',
+            'app.bundle': [ '@babel/polyfill', './app.js' ],
```

Comment:
polyfill is for async/await? We're going to start using that? (If so I'll add it to spot as well.) 

Also, should we try to target the async/away polyfills and not bring in the package? Admittedly, I haven't looked into it myself. Another concern I have would be how to get the polyfills to the other web bundles below. I'm also tempted to recommend creating yet another bundle so the bundles below can import the polyfills through a script tag.

Reply:
Yes, it was needed for async/await. I hope we can get rid of some additional manual polyfills with it as well, but it's not the scope of this PR

Reply:
I'd say it's well in scope of this PR. We have removed babel polyfill previously and I don't want it back. Let's look at the options. There seem to be a number of them. I'm going to try some.

Reply:
I'm wondering if it is better to introduce async/await trough separate PR. This way you can refactor in new async/await PR the other places of the code that you mentioned. In this PR you are using it only at one place and we will have to add the polyfill dependancy. I'm not against starting using async/await but in this case it feels a little bit that you actually don't really need it in the PR but you have added it only to be able to use it later....

Reply:
@saghul I'm waiting for your suggestions around this. Otherwise I'd go with this implementation.

@hristoterezov Yeah, you're probably right, although I did all this "cleaning up" to have better readability, that involves async/await, If you feel a strong urge to have it in a separate commit, I'm happy to do it, but I'd be happier to leave it like this, because then I had to implement another logic here with promises :)

Reply:
@zbettenbuk I'll look into it, but you should too.

@hristoterezov Adding async/await was the heart of this PR, adding support for it without a single usage would be weird IMHO.

Reply:
Ok, I found a solution! The trick is to let Babel know what browsers we support: https://github.com/saghul/jitsi-meet/blob/8f056c278bc5e6c23fd146cb1296354f5e13c9f8/webpack.config.js#L61-L68

Then we can use async natively.

Here is a commit which removes the polyfill and uses that approach: https://github.com/saghul/jitsi-meet/commit/8f056c278bc5e6c23fd146cb1296354f5e13c9f8